modules/tensor_mechanics/doc/content/source/nodalkernels/NodalGravity.md

```diff
@@ -0,0 +1,13 @@
+# Nodal Gravity
+
+!syntax description /NodalKernels/NodalGravity
+
+# Description
+
+This NodalKernel computes the gravitational force proportional to nodal mass in the coordinate direction corresponding to the assigned variable. A constant gravitational acceleration can be provided using `gravity_value` parameter and an acceleration varying in space and time can be provided using the `function` parameter. A constant mass for all the nodes in the given boundary can be provided using the `mass` parameter. Otherwise, a CSV file containing nodal positions and the corresponding nodal masses can also be provided using the `nodal_mass_file` parameter.
+
```

Comment:
Add some explanation about the formatting of the nodal mass file (maybe a simple example provided via a link to one of the test cases would be helpful).